core/src/main/java/com/orange/ouds/core/theme/OudsTheme.kt

```diff
@@ -61,6 +65,20 @@ object OudsTheme {
         Invert, ForceDark, ForceLight
     }
 
+    /**
+     * The theme settings.
+     *
+     * @property roundedCorners Indicates if rounded corners should be applied to several components, such as [OudsButton].
+     *   Set to `false` for a default finish, or `true` for a finish with rounded corner.
+     *   To be favored in more emotional, immersive contexts or those tied to specific visual identities.
+     *   For standard or business-oriented journeys, keep the default corners.
+     *   This evolution addresses the need for flexibility in adapting the design to certain brand contexts.
+     *   Please note that this setting may have no effect depending on the theme used.
+     * @constructor Creates an instance of [OudsTheme.Settings].
+     */
+    @Parcelize
+    data class Settings(val roundedCorners: Boolean) : Parcelable
```

Comment:
As discussed together, we should have a specific rounded corner setting for each component that allows rounded corners. Here we can rename the property `buttonRoundedCorners` and adapt the documentation.